packages/react/src/Testimonial/Testimonial.module.css

```
@@ -18,16 +23,39 @@
 }
 
 .Testimonial--size-large {
+  font-size: var(--brand-Testimonial-quote-fontSize-large);
+  font-weight: var(--brand-Testimonial-quote-fontWeight-large);
+  letter-spacing: var(--brand-Testimonial-quote-letterSpacing-large);
+  line-height: var(--brand-Testimonial-quote-lineHeight-large);
   gap: var(--base-size-24);
 }
 
+.Testimonial--size-large::before {
+  font-size: 3em;
+  line-height: 1;
+}
+
 .Testimonial-quote {
+  font-size: 1em;
+  font-family: var(--brand-fontStack-sansSerif);
   margin-top: calc(var(--base-size-16) * -1); /* negate grid gap */
   display: block;
 }
 
 .Testimonial--size-large .Testimonial-quote {
-  font-size: var(--brand-text-size-700);
+  margin-top: calc(var(--base-size-24) * -1); /* negate grid gap */
+}
+
+.Testimonial-quote--muted {
+  color: var(--brand-color-text-subtle);
+}
```

referring to this [comment](https://github.com/primer/brand/pull/372/files#r1298600302) we might not need this styles. 

Anyways the color should be `muted`, as well as the SectionIntro or the Timeline `em` color implementations

```suggestion
.Testimonial-quote--muted {
  color: var(--brand-color-text-muted);
}
```